resource: file support for SAML is broken

The path to the file is not properly computed when using the `resource:` prefix.

```
Caused by: org.pac4j.saml.exceptions.SAMLException: Could not create keystore
	at org.pac4j.saml.client.SAML2ClientConfiguration.createKeystore(SAML2ClientConfiguration.java:472)
	at org.pac4j.saml.client.SAML2ClientConfiguration.<init>(SAML2ClientConfiguration.java:131)
	at org.pac4j.saml.client.SAML2ClientConfiguration.<init>(SAML2ClientConfiguration.java:95)
	at org.pac4j.config.client.PropertiesConfigFactory.tryCreateSaml2Client(PropertiesConfigFactory.java:223)
	at org.pac4j.config.client.PropertiesConfigFactory.build(PropertiesConfigFactory.java:115)
	at org.apereo.cas.support.pac4j.config.Pac4jConfiguration.builtClients(Pac4jConfiguration.java:218)
....
Caused by: java.io.FileNotFoundException: /Users/bidou/sources/cas-pac4j-oauth-demo/ile:/Users/bidou/sources/cas-pac4j-oauth-demo/cas-pac4j-oauth-client-demo/target/cas.war!/WEB-INF/classes!/samlKeystore.jks (No such file or directory)
	at java.io.FileOutputStream.open0(Native Method)
	at java.io.FileOutputStream.open(FileOutputStream.java:270)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:213)
```

--- a/docs/cas-server-documentation/installation/Configuration-Properties.md
+++ b/docs/cas-server-documentation/installation/Configuration-Properties.md
@@ -1556,7 +1556,7 @@ Delegate authentication to an external OpenID Connect server.
 
 ### SAML
 
-Delegate authentication to an external SAML2 IdP.
+Delegate authentication to an external SAML2 IdP (do not use the `resource:` or `classpath:` prefixes for the `keystorePath` property).
 
 ```properties
 # cas.authn.pac4j.saml.keystorePassword=

